### DISCUSSION TITLE
Change Temporarily Inactive title to Inactive

### DISCUSSION THREAD

**Discussion by Grateful-Viv** [code4recovery/12-step-meeting-list#701] [Ideas]
Would it be possible to change where it says **Temporarily Inactive** to **Inactive**? At this point, I think it's beyond being temporarily.

This came up today because we had a meeting discontinue, but rather than remove it, I wanted to display it for a period of time in case people are still looking for it and wondering why it disappeared.

**Comment by joshreisner** [comment]
i support this, this is what TSML UI does 👍

**Comment by brianw-area24** [comment]
At first I wasn't sure about this, as my use case is quite a bit different, in that we've had in the past, and I expect we'll have some in March, where a meeting doesn't meet for a week or two, due to something like other big events happening at the same time, or temporarily not able to meet because a location isn't available. This is way more common in my district than a meeting discontinuing.

However, as I thought more about this, the more generic "Inactive" does cover be both cases, without needing the descriptor "Temporarily". In either case, the user should probably click to see the details page, and may find something in the description field. Therefore, I'm OK with this suggestion also.
